data/json/items/armor/jewelry.json

```diff
@@ -1111,7 +1111,381 @@
     "symbol": ",",
     "color": "yellow",
     "use_action": [ "MEDITATE" ],
-    "flags": [ "NO_WEAR_EFFECT" ]
+    "flags": [ "NO_WEAR_EFFECT" ],
+    "variants_type": "generic",
+    "variants": [
+      {
+        "id": "lutheran_necklace",
+        "name": { "str": "Lutherian Seal necklace" },
+        "description": "A necklace made of a fine gold chain with the Luther Seal symbol as a pendant.  It is a symbol of the Lutheran Christian faith: The cross mean that humanity was saved from sin because Jesus died on the cross, the heart means that humanity live because Jesus shed his blood for it, and the flower means that humanity have peace from our faith, and the background means that we have hope that we will be with Jesus in heaven one day.",
```

Comment:
```suggestion
        "description": "A necklace made of a fine gold chain with the Luther Seal symbol as a pendant.  It is a symbol of the Lutheran Christian faith: The cross mean that humanity was saved from sin because Jesus died on the cross, the heart means that humanity lives because Jesus shed his blood for it, and the flower means that humanity has peace from its faith, and the background means that they have hope that they will be with Jesus in heaven one day.",
```